pandas/core/reshape/tile.py

```diff
@@ -24,53 +24,64 @@
 def cut(x, bins, right=True, labels=None, retbins=False, precision=3,
         include_lowest=False):
     """
-    Return indices of half-open bins to which each value of `x` belongs.
+    Return indices of half-open `bins` to which each value of `x` belongs.
```

Comment:
I know it was already there, but I wondering if we can make this first sentence better. Because I have to say I have to read it very carefully to actually understand it :)

Some ideas:
 
- "Return indices" is actually not correct? As it just returns the intervals or bins itself?
- Looking at https://en.wikipedia.org/wiki/Discretization_of_continuous_features and https://en.wikipedia.org/wiki/Data_binning so maybe something with the terms "convert continuous values in discrete bins" ?